cps/src/main/resources/org/jenkinsci/plugins/workflow/cps/Snippetizer/dslReferenceContent.groovy

```diff
@@ -12,47 +12,64 @@ Snippetizer snippetizer = my;
 def l = namespace(lib.LayoutTagLib)
 def st = namespace("jelly:stapler")
 
-
-h1(_("Steps"))
-for (StepDescriptor d : snippetizer.getStepDescriptors(false)) {
-    generateStepHelp(d);
+st.adjunct(includes: 'org.jenkinsci.plugins.workflow.cps.Snippetizer.css.workflow')
+
+h1(_("DSL Reference"))
+div(class:'steps-box basic'){
+	h2(_("Steps"))
+	dl(class:'steps basic root'){
+		for (StepDescriptor d : snippetizer.getStepDescriptors(false)) {
+		    generateStepHelp(d);
+		}
+	}
 }
 
-h1(_("Advanced/Deprecated Steps"))
-for (StepDescriptor d : snippetizer.getStepDescriptors(true)) {
-    generateStepHelp(d);
+div(class:'steps-box advanced'){
+	h2(_("Advanced/Deprecated Steps"))
+	dl(class:'steps advanced root'){
+		for (StepDescriptor d : snippetizer.getStepDescriptors(true)) {
+		    generateStepHelp(d);
+		}
+	}
 }
 
-h1(_("Variables"))
-
-for (GlobalVariable v : snippetizer.getGlobalVariables()) {
-    h2 {
-        code(v.getName())
-    }
-    RequestDispatcher rd = request.getView(v, "help");
-    div(class:"help", style:"display: block") {
-        if (rd != null) {
-            st.include(page: "help", it: v)
-        } else {
-            raw("(no help)")
-        }
-    }
+div(class:'steps-box variables'){
+
+	h2(_("Variables"))
+	dl(class:'steps variables root'){
+		for (GlobalVariable v : snippetizer.getGlobalVariables()) {
+		    dt {
+		        code(v.getName())
+		    }
+		    RequestDispatcher rd = request.getView(v, "help");
+		    dd{
+			    div(class:"help", style:"display: block") {
+			        if (rd != null) {
+			            st.include(page: "help", it: v)
+			        } else {
+			            raw("(no help)")
+			        }
+			    }
+			}
+		}
+	}
 }
 
-
 def generateStepHelp(StepDescriptor d) throws Exception {
     return {
-        h2 {
+    	dt(class:'step-title'){
```

Comment:
The step titles are now the same size as the parameter titles, and feel too big in general. Might make sense to shrink both, and have different sizes for nested titles again?